kibana/kibana.yml

```diff
@@ -1,67 +1,79 @@
 # Kibana is served by a back end server. This controls which port to use.
-port: 5601
+# server.port: 5601
 
 # The host to bind the server to.
-host: "0.0.0.0"
+# server.host: "0.0.0.0"
+
+# If you are running kibana behind a proxy, and want to mount it at a path,
+# specify that path here. The basePath can't end in a slash.
+# server.basePath: ""
+
+# The maximum payload size in bytes on incoming server requests.
+# server.maxPayloadBytes: 1048576
 
 # The Elasticsearch instance to use for all your queries.
-elasticsearch_url: "https://es_host:es_port"
+elasticsearch.url: https://es_host:es_port
 
 # preserve_elasticsearch_host true will send the hostname specified in `elasticsearch`. If you set it to false,
 # then the host you use to connect to *this* Kibana instance will be sent.
-elasticsearch_preserve_host: true
+# elasticsearch.preserveHost: true
 
 # Kibana uses an index in Elasticsearch to store saved searches, visualizations
 # and dashboards. It will create a new index if it doesn't already exist.
-kibana_index: ".kibana"
+# kibana.index: ".kibana"
+
+# The default application to load.
+# kibana.defaultAppId: "discover"
 
-# If your Elasticsearch is protected with basic auth, this is the user credentials
-# used by the Kibana server to perform maintence on the kibana_index at statup. Your Kibana
-# users will still need to authenticate with Elasticsearch (which is proxied thorugh
+# If your Elasticsearch is protected with basic auth, these are the user credentials
+# used by the Kibana server to perform maintenance on the kibana_index at startup. Your Kibana
+# users will still need to authenticate with Elasticsearch (which is proxied through
 # the Kibana server)
-kibana_elasticsearch_username: kibana
-kibana_elasticsearch_password: changeme
+elasticsearch.username: "kibana"
+elasticsearch.password: "changeme"
```

Comment:
it's a tiny thing but could we change this :)

something like "dummy" or "foo" or... could this bit just be left out?